src/main/java/views/GamePane.java

```diff
@@ -0,0 +1,22 @@
+package views;
+
+import javafx.scene.layout.BorderPane;
+
+
+public abstract class GamePane extends BorderPane {
+
+    /**
+     * Connects all components into the root of a javafx.scene.Scene.
+     */
+    abstract void connectComponents();
+
+    /**
+     * Styles all components as required.
+     */
+    abstract void styleComponents();
+
+    /**
+     * Sets callbacks for all interactive components.
+     */
+    abstract void setCallbacks();

```

Comment:
Good use of comments. Explains what the functions do.

Reply:
Good Comments, I also think that it would help if other parts of the code were explained like this.